ckanext/data_qld_theme/templates/package/snippets/additional_info.html

```diff
@@ -12,6 +12,7 @@
 
 {%- if h.data_qld_user_has_admin_access(True) == False -%}
   {%- do exclude_fields.append('author_email') -%}
+  {%- do exclude_fields.append('next_update_due') -%}
```

Comment:
Can any user with editor or admin access to any organisation view this metadata field or is it only users with editor/admin access to this datasets organisation?
If it is the later do a check like `data_qld_show_de_identified_data`